app/src/main/java/com/keylesspalace/tusky/view/BackgroundMessageView.kt

```diff
@@ -25,6 +26,9 @@ class BackgroundMessageView @JvmOverloads constructor(
     private val binding = ViewBackgroundMessageBinding.inflate(LayoutInflater.from(context), this)
 
     init {
+        // TODO this seems to be needed? Otherwise most parts are hidden?
+        // But why? This is exactly the same as specified in the xml.
```

Comment:
Because it is `tools:gravity` and `tools:orientation`, the tools namespace is only used for previews (e.g. in Android Studio). I don't think it can be done differently, this is fine.